Dockerfile.template

```diff
@@ -6,13 +6,12 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node#release-team
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
```

Comment:
It does still exist: https://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0x9554F04D7259F04124DE6B476D5A82AC7E37093B

(But, it appears that Chris only signed "past releases", so maybe safe to remove anyhow?)

Reply:
I've been getting errors like "no data" from the keyservers for this key. I assumed this was the problem. Having said that, I've retried a few times, and am not getting those errors any more.

Reply:
I think it might be safe to remove it but I'd like to hear from the rest of the team.

And if they are ok with it, the other *.template files should have the same change.